### PR TITLE
patch messages definied enums

### PR DESCRIPTION
**Pull request by Adphi**

### CODE REVIEW

**Comment by ydnar** [comment]
Nice catch, thanks! Can you add relevant tests for this?

**Comment by Adphi** [comment]
I'll take a look at the tests.

**Comment by Adphi** [comment]
I'm not sure about the tests' implementation, is it ok for you ?

File: tests/enum/enum_test.go [hunk 1]
```diff
@@ -26,6 +26,21 @@ func TestRenamedEnumValue(t *testing.T) {
 	}
 }
 
+func TestRenamedInnerEnumValue(t *testing.T) {
```
Comment (ydnar) [inline tests/enum/enum_test.go:29]: The other tests use `Nested` instead of `Inner` here, but otherwise looks OK.

**Review by ydnar — APPROVED** [review]

**Pull request merged**